python/ray/experimental/channel/torch_tensor_nccl_channel.py

```diff
@@ -648,7 +650,7 @@ def _init_nccl_group(
         [actor.__ray_call__.remote(_do_check_has_gpu) for actor in actors]
     )
     for has_gpu, actor in zip(has_gpus, actors):
-        if not has_gpu:
+        if not has_gpu and not isinstance(custom_nccl_group, CPUNcclGroup):
```

Comment:
I think this also needs a check that `custom_nccl_group is not None`?

Reply:
fixed